packages/web-components/src/radio/radio.ts

```diff
@@ -1,122 +1,80 @@
-import { observable } from '@microsoft/fast-element';
-import { keySpace } from '@microsoft/fast-web-utilities';
-import type { RadioGroup } from '../radio-group/index.js';
-import type { StaticallyComposableHTML } from '../utils/template-helpers.js';
-import { FormAssociatedRadio } from './radio.form-associated.js';
-
-/**
- * @public
- */
-export type RadioControl = Pick<HTMLInputElement, 'checked' | 'disabled' | 'focus' | 'setAttribute' | 'getAttribute'>;
-
-/**
- * Radio configuration options
- * @public
- */
-export type RadioOptions = {
-  checkedIndicator?: StaticallyComposableHTML<Radio>;
-};
+import { BaseCheckbox } from '../checkbox/checkbox.js';
 
 /**
  * A Radio Custom HTML Element.
- * Implements the {@link https://www.w3.org/TR/wai-aria-1.1/#radio | ARIA radio }.
+ * Implements the {@link https://w3c.github.io/aria/#radio | ARIA `radio` role}.
  *
- * @slot checked-indicator - The checked indicator
- * @slot - The default slot for the label
- * @csspart control - The element representing the visual radio control
- * @csspart label - The label
+ * @slot checked-indicator - The checked indicator slot
  * @fires change - Emits a custom change event when the checked state changes
+ * @fires input - Emits a custom input event when the checked state changes
  *
  * @public
  */
-export class Radio extends FormAssociatedRadio implements RadioControl {
+export class Radio extends BaseCheckbox {
   /**
-   * The name of the radio. See {@link https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#htmlattrdefname | name attribute} for more info.
-   */
-  @observable
-  public name!: string;
-
-  /**
-   * The element's value to be included in form submission when checked.
-   * Default to "on" to reach parity with input[type="radio"]
+   * Indicates the position of the radio in a radio group.
    *
-   * @internal
+   * @public
+   * @remarks
+   * Reflects the {@link https://developer.mozilla.org/docs/Web/API/ElementInternals/ariaPosInSet | `ElementInternals.ariaPosInSet`} property.
    */
-  public initialValue: string = 'on';
+  public get ariaPosInSet(): string | null {
```

Comment:
I think the ARIA Reflection properties are supposed to reflect the ARIA attributes set on the element itself (which we should get for free in supported browsers), instead of the ones set on element internals.

Reply:
I removed these getters/setters for these ARIA properties in favor of setting the properties directly on the element.